packages/fluentui/accessibility/src/behaviors/Tree/treeTitleBehavior.ts

```diff
@@ -36,7 +36,7 @@ const treeTitleBehavior: Accessibility<TreeTitleBehaviorProps> = props => ({
       },
       ...(props.selectable && {
         performClick: {
-          keyCombinations: [{ keyCode: keyboardKey.Spacebar }],
+          keyCombinations: [{ keyCode: keyboardKey.Spacebar }, { keyCode: keyboardKey.Enter }],
```

Comment:
adding Enter key will do selection as well on Enter, and this we wouldn't like

Reply:
Removed